features/other_a/hit_counts.feature

```diff
@@ -0,0 +1,129 @@
+@javascript
+Feature: Hit Counts
+  Background:
+    Given I limit myself to the Archive
+
+  # Throughout these tests, we use the "all hit count information is reset"
+  # step because logging in/logging out may result in the user being redirected
+  # to the page that the user was just on, i.e. the work whose hit count we're
+  # trying to test.
+  #
+  # We also use the "I wait 1 second" step to make sure that the browser has
```

Comment:
I was a little worried about how this would play out with the two sequential AJAX requests you get when you're logged out, but I've run the tests 10+ times locally and haven't had a test failure, so I think it's good.🤞

Reply:
Did you get failures locally before? My dev environment is slow enough that I never get these flaky failures there, I have to use Travis.

Reply:
I usually have to use Travis, but in this case I actually did get failures locally.